semantics/cpp14/language/common/typing.k

```diff
@@ -277,6 +278,8 @@ module CPP-TYPING-SYNTAX
 
      syntax CPPType ::= implicitObjectParameterType(CPPFunctionType) [function]
 
+     syntax LVal ::= getBaseClassSubobject(base: LVal, baseType: CPPClassType) [strict]
```

Comment:
you're making this a constructor for the LVal sort, which is likely to cause huge problems. The only constructors for LVal should be le and lv.